ta/secure_key_services/src/processing_ec.c

```diff
@@ -1274,6 +1275,16 @@ uint32_t generate_ec_keys(struct sks_attribute_head *proc_params,
 
 	TEE_InitValueAttribute(&tee_key_attr[0], TEE_ATTR_ECC_CURVE,
 				tee_curve, 0);
+	tee_count++;
+
+	/* pass the label to the private exponent */
+	rv = get_attribute_ptr(*pub_head, SKS_CKA_LABEL, &b_ptr, &b_size);
+
+	if (rv == SKS_OK && b_size == 11 && !memcmp(b_ptr, "SE_", 3)) {
+		TEE_InitRefAttribute(&tee_key_attr[tee_count],
+				     TEE_ATTR_ECC_PRIVATE_VALUE, b_ptr, b_size);
+		tee_count++;
+	}
 
```

Comment:
Can you protect this extra logic with a ifdef or similar that only gets applied if optee-os was built with se050 support?

Reply:
Prefer `if (IS_ENABLED(CFG_SE05_...))`. Requires `#include <config.h>`.

Reply:
ok